lib/ddtrace/configuration/settings.rb

```diff
@@ -44,6 +44,39 @@ def initialize(*_)
         tags
       end
 
+      # {https://docs.datadoghq.com/agent/ Datadog Agent} configuration.
+      # @configure_with {Datadog}
+      # @public_api
+      settings :agent do
+        # @configure_with {Datadog}
+        # @default `DD_AGENT_HOST` environment variable, otherwise `127.0.0.1`
+        # @return [String,nil]
+        option :host
+
+        # Configuration for trace agent access.
+        # @configure_with {Datadog::Tracing}
+        # @public_api
+        settings :tracer do
+          # Trace agent port.
+          # @configure_with {Datadog::Tracing}
+          # @default `DD_AGENT_HOST` environment variable, otherwise `8126`
+          # @return [String,nil]
+          option :port
```

Comment:
```suggestion
          # @default `DD_TRACE_AGENT_PORT` environment variable, otherwise `8126`
          # @return [String,nil]
          option :port
```